### PR TITLE
Updated the db.js file

### PR DESCRIPTION
**Pull request by Sarvesh025**

### CODE REVIEW

File: _helpers/db.js [hunk 1]
```diff
@@ -24,9 +24,103 @@ async function initializeDb() {
...
+    db.CouponOrder = require('../_models/CouponOrder')(sequelize, DataTypes);
+    db.CouponSetting = require('../_models/CouponSetting')(sequelize, DataTypes);
+    db.Currency = require('../_models/Currency')(sequelize, DataTypes);
+    db.CurrentTheme = require('../_models/CurrentTheme')(sequelize, DataTypes);
+    db.CustomCssJs = require('../_models/CustomCssJs')(sequelize, DataTypes);
+    db.CustomerAddressBook = require('../_models/CustomerAddressBook')(sequelize, DataTypes);
+    db.CustomerOrderAmount = require('../_models/CustomerOrderAmount')(sequelize, DataTypes);
+    db.Customers = require('../_models/Customers')(sequelize, DataTypes);
+    db.DefaultAccounts = require('../_models/DefaultAccounts')(sequelize, DataTypes);
+    // db.demo.model = require('../_models/demo.model')(sequelize, DataTypes);
```
Comment (arijeet-mukherjee) [inline _helpers/db.js:50]: Why there is commented code here ? Remove this model import comment if not required.

File: _helpers/db.js [hunk 2]
```diff
@@ -24,9 +24,103 @@ async function initializeDb() {
...
+    db.DemoSettings = require('../_models/DemoSettings')(sequelize, DataTypes);
+    db.EmailTemplateSettings = require('../_models/EmailTemplateSettings')(sequelize, DataTypes);
+    db.FailedJobs = require('../_models/FailedJobs')(sequelize, DataTypes);
+    db.Gallery = require('../_models/Gallery')(sequelize, DataTypes);
+    db.GalleryDetail = require('../_models/GalleryDetail')(sequelize, DataTypes);
+    db.GalleryTags = require('../_models/GalleryTags')(sequelize, DataTypes);
+    db.HomeBanners = require('../_models/HomeBanners')(sequelize, DataTypes);
+    db.Languages = require('../_models/Languages')(sequelize, DataTypes);    
     db.LeakCred = require('../_models/LeakCred')(sequelize, DataTypes);
+    // db.leakcred.model = require('../_models/leakcred.model')(sequelize, DataTypes);
```
Comment (arijeet-mukherjee) [inline _helpers/db.js:60]: Here also a commented code

File: _helpers/db.js [hunk 3]
```diff
@@ -24,9 +24,103 @@ async function initializeDb() {
...
     db.LeakCredDetail = require('../_models/LeakCredDetail')(sequelize, DataTypes);
+    // db.leakcreddetail.model = require('../_models/leakcreddetail.model')(sequelize, DataTypes);
```
Comment (arijeet-mukherjee) [inline _helpers/db.js:62]: Here too

File: _helpers/db.js (hunk 1 above)
Comment (Sarvesh025) [reply _helpers/db.js:50]: Ayush bhaiya told to keep it commented, should i delete and raise pr again?

**Pull request closed without merging**